tracer/src/Datadog.Trace/AppSec/VulnerabilitiesModule.cs

```diff
@@ -0,0 +1,42 @@
+// <copyright file="VulnerabilitiesModule.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using Datadog.Trace.AppSec.Rasp;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.Iast;
+
+#nullable enable
+
+namespace Datadog.Trace.AppSec;
+
+internal static class VulnerabilitiesModule
+{
+    internal static void OnPathTraversal(string path)
+    {
+        if (!string.IsNullOrEmpty(path))
+        {
+            IastModule.OnPathTraversal(path);
+            RaspModule.OnLfi(path);
+        }
+    }
+
+    internal static void OnSSRF(string uriText)
+    {
+        if (!string.IsNullOrEmpty(uriText))
+        {
+            IastModule.OnSSRF(uriText);
+            RaspModule.OnSSRF(uriText);
+        }
+    }
+
+    internal static void OnSqlI(string query, IntegrationId integrationId)
```

Comment:
minor nit: not a fan of `OnSqlI` - would prefer `OnSqlInjection`, or even better `OnSqlQuery`. The latter seems better as it describes when to call it, but Iast and Rasp could feasibly do _other_ checks too?

Definitely could be another PR though

Reply:
I agree. Also, it should be consistent with the IAST naming. Done!